integration-testing/src/main/proto/io/grpc/testing/integration/test.proto

```diff
@@ -58,6 +58,11 @@ service TestService {
   rpc StreamingInputCall(stream StreamingInputCallRequest)
       returns (StreamingInputCallResponse);
 
+  // Start uploading only after receiving a confirmation message from the server.
+  // The server returns the aggregated size of client payload as the final result.
+  rpc CheckedUploadCall(stream StreamingInputCallRequest)
```

Comment:
Why "Checked"? And "Upload" seems _yet another_ name for client streaming.

Could we name it "DelayedStreamingInputCall" or something more along those lines, since it is only a small modification from StreamingInputCall?